src/resources/views/partials/character.blade.php

```diff
@@ -1,17 +1,17 @@
 @if (request('all_linked_characters') === "true")
-  {!! img('character', $character_id, 32, ['class' => 'img-circle eve-icon small-icon'],false) !!}
+  {!! img('character', $character_id, 32, ['class' => 'img-circle eve-icon small-icon'], false) !!}
 @endif
 
 @if (isset($character->name) )
```

Comment:
should be `@if (isset($character->name))`

Reply:
yes and mergeconflict resolved